src/gluonts/support/util.py

```diff
@@ -470,3 +471,18 @@ def erf(F, x: Tensor):
 
     res = ones - t * (inner - 1.26551223 - x.square()).exp()
     return F.where(F.broadcast_greater_equal(x, zeros), res, -1.0 * res)
+
+
+def get_download_path() -> Path:
+    """
+
+    Returns
+    -------
+    Path
+        default path to download datasets, models of gluon-ts, for instance /home/username/.mxnet/gluon-ts/
```

Comment:
Minor comment, but I think this line exceeds the 79 chars limit (black does not enforce auto line-wrap at the moment).